src/python/ensembl/utils/__init__.py

```diff
@@ -0,0 +1,15 @@
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Utils module"""
+from .rloader import *
```

Comment:
PEP8 is your friend:
```suggestion

from .rloader import *
```

Reply:
I would tend to believe that PyCharm not being reporting this as a pep8 issue, this is not?

Reply:
The only PEP8 reference I found was: "You should put a blank line between each group of imports."  IMO it's nice to have a blank line between module docstring and imports but not a must. Seems more a pylint thing.

Reply:
`Should` is not `must` :-)